firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/FirebaseCrashlyticsReportManager.java

```diff
@@ -148,4 +174,23 @@ private boolean onReportSendComplete(Task<CrashlyticsReport> task) {
     // TODO: Something went wrong. Log? Throw?
     return false;
   }
+
+  private static List<CustomAttribute> getSortedCustomAttributes(Map<String, String> attributes) {
+    if (attributes == null || attributes.isEmpty()) {
+      return null;
+    }
+
+    List<CustomAttribute> attributesList = new ArrayList<>();
```

Comment:
Since this is a copy, you might consider setting a capacity, e.g., `new ArrayList<>(attributes.size())`?

Reply:
:+1: